nbs/introduction.ipynb

```diff
@@ -14,6 +14,16 @@
     "In this tutorial you will learn the basics of spanner workbench"
```

Comment:
<p>When you open this notebook in colab, make it so this code can actually run to install the workbench (By making it a code cell with a <code>!</code> prefix.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>